featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -686,11 +686,11 @@ def parallel_calculate_chunks(cutoff_time, chunk_size, feature_set, approximate,
     except Exception:
         raise
     finally:
-        if client is not None:
-            client.close()
-
-        if 'cluster' not in dask_kwargs and cluster is not None:
-            cluster.close()
+        if 'cluster' not in dask_kwargs:
+            if cluster is not None:
+                cluster.close()
+            if client is not None:
+                client.close()
```

Comment:
in the past we swapped the order to close the client before the cluster so that there wouldn't be these benign warnings from the client about losing contact with the cluster

Reply:
Deleted this code change for now - will revisit later if needed.